source/_integrations/flux.markdown

```diff
@@ -6,6 +6,7 @@ ha_category:
 ha_release: 0.21
 ha_quality_scale: internal
 ha_domain: flux
+ha_iot_class: ~
```

Comment:
```suggestion
ha_iot_class: Calculated
```

Reply:
I wasn't sure which one it would fit best but this probably would make more sense